yellowbrick/cluster/silhouette.py

```diff
@@ -152,6 +152,7 @@ def draw(self, labels):
         colors = color_palette(self.colormap, self.n_clusters_)
 
         # For each cluster, plot the silhouette scores
+        self.y_tick_pos = []
```

Comment:
Since this is a "learned attribute" (any data that is created when `fit()` is called), let's append an underscore, `self.y_tick_pos_` 

Would you also please document this attribute in the docstring of the class? See [icdm.py Line 123](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/cluster/icdm.py#L123) for an example of how to document the learned attributes.